.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### Please paste your output of croutonversion from chroot below:
+```
+Replace this line with your output.
```

Comment:
That's a nice touch.  It seems like we could get rid of the heading and just say "Please paste the output of the following command here: sudo edit-chroot -al"